test/Ionide.ProjInfo.Tests/Tests.fs

```diff
@@ -955,30 +948,77 @@ let debugTets toolsPath =
             )
 
 let tests toolsPath =
+    let testSample3WorkspaceLoaderExpected =
+        [ ExpectNotification.loading "c1.fsproj"
+          ExpectNotification.loading "l1.csproj"
+          ExpectNotification.loaded "l1.csproj"
+          ExpectNotification.loading "l2.fsproj"
+          ExpectNotification.loaded "l2.fsproj"
+          ExpectNotification.loaded "c1.fsproj" ]
+
+    let testSample3GraphExpected =
+        [ ExpectNotification.loading "c1.fsproj"
+          ExpectNotification.loaded "c1.fsproj" ]
+
+    let testSlnExpected =
+        [ ExpectNotification.loading "c1.fsproj"
+          ExpectNotification.loading "l2.fsproj"
+          ExpectNotification.loaded "l2.fsproj"
+          ExpectNotification.loaded "c1.fsproj"
+          ExpectNotification.loading "l1.fsproj"
+          ExpectNotification.loaded "l1.fsproj"
+          ExpectNotification.loaded "l2.fsproj" ]
+
+    let testSlnGraphExpected =
```

Comment:
We won't get duplicate messages anymore so I had to pivot on expectations for either implementation